### PR TITLE
Componentize wash with wascap

### PR DESCRIPTION
**Pull request by ricochet**
## Feature or Problem

Bump the version of wascap that is componentized and backward compatible.

## Release Information

Next release

## Consumer Impact

This isn't breaking and is backwards compatible.

## Testing

I've only run unit tests and locally run wash claims to get/set claims. Then I loaded that component into a component enabled wasmCloud-OTP host.

Built on platform(s)
- [ ] x86_64-linux
- [ ] aarch64-linux
- [ ] x86_64-darwin
- [x] aarch64-darwin
- [ ] x86_64-windows

Tested on platform(s)
- [ ] x86_64-linux
- [ ] aarch64-linux
- [ ] x86_64-darwin
- [x] aarch64-darwin
- [ ] x86_64-windows

### CODE REVIEW

**Comment by ricochet** [comment]
Making a draft PR for visibility but leaving as a draft until we release wascap and have a version to pin to.

File: Cargo.toml [hunk 1]
```diff
@@ -133,7 +133,8 @@ tokio-stream = "0.1"
 tokio-tar = "0.3"
 toml = "0.5"
 walkdir = "2.3"
-wascap = "0.9.2"
+# wascap = "0.9.2"
+wascap = { git = "https://github.com/wasmCloud/wascap" }
```
Comment (brooksmtownsend) [inline Cargo.toml]: I think 0.10.0 is out!

**Review by brooksmtownsend — DISMISSED** [review]
Woo no code changes!

**Review by connorsmith256 — APPROVED** [review]
Woohoo

**Review by autodidaddict — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**